.github/ISSUE_TEMPLATE/libmamba-bug-report.yml

```
@@ -0,0 +1,84 @@
+
+        There will always be more issues than there is time to do them, and so we will need to selectively close issues that don't provide enough information, so we can focus our time on helping people like you who fill out the issue form completely. Thank you for your collaboration!
+
+        There are also already a lot of open issues, so please take 2 minutes and search through existing ones to see if what you are experiencing already exists
+
+        Thanks for helping conda be amazing. We are nothing without people like you helping build a better community 💐!
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Checklist
+      description: Please confirm and check all the following options.
+      options:
+        - label: I added a descriptive title
+          required: true
+        - label: I searched for other issues and couldn't find a solution or duplication
+          required: true
+        - label: I looked at the [README.md](https://github.com/conda-incubator/conda-libmamba-solver) and didn't see anything to help
+          required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: And what should have happened instead? This really helps us triage quicker and greatly increases the chance that we can get around to solve your issue
+      placeholder: Tell us what you see!
+      value: "A wild Caterpie appeared!"
+    validations:
+      required: true
+  - type: textarea
+    id: info
+    attributes:
+      label: Conda info
+      description: |
+        Let's get the info on your conda setup. Run the following command from your command line and paste the output
+
+        ```bash
+        conda info
+        ```
+      render: shell
+  - type: textarea
+    id: config
+    attributes:
+      label: Conda config
+      description: |
+        Let's see what your conda config looks like. Run the following command from your command line and paste the output
+
+        ```bash
+        conda config --show-sources
+        ```
+      render: shell
+  - type: textarea
+    id: list
+    attributes:
+      label: Conda list
+      description: |
+        These are the packages installed in your conda environment and the conda channels used. They tell us where you are pulling packages from. Make sure you are in the environment where \n you are seeing the issues. Run the following command from your command line and paste the output
+
+        ```bash
+        conda list --show-channel-urls
+        ```
+      render: shell
+
+  - type: textarea
+    attributes:
+      label: Log file
+      description: |
+        Please attach the associated log file (drag and drop the file into the textbox
+        below). The log file can be found at `$CONDA_PREFIX\pkgs\.logs\`. `$CONDA_PREFIX` is the path to the libmamba environment. The
+        exact path is also printed to the command line when running commands with
+        `--experimental-solver=libmamba`.
```

```suggestion
        the experimental libmamba solver enabled.
```
There are a few other way to enable the solver, so I think we can be a little less specific here.

---

Done